openwisp_controller/config/tests/test_views.py

```diff
@@ -103,12 +103,36 @@ def test_get_default_templates_with_backend_filtering(self):
         self.assertIn(str(t1.pk), templates)
         self.assertNotIn(str(t2.pk), templates)
 
-    def test_get_default_templates_403(self):
+    def test_get_default_templates_authorization(self):
         org1 = self._create_org(name='org1')
-        response = self.client.get(
-            reverse('admin:get_default_templates', args=[org1.pk])
-        )
-        self.assertEqual(response.status_code, 403)
+        with self.subTest('Unauthenticated user'):
+            response = self.client.get(
+                reverse('admin:get_default_templates', args=[org1.pk])
+            )
+            self.assertEqual(response.status_code, 403)
+
+        with self.subTest('Authenticated non-staff user'):
+            user = self._create_user()
+            self.client.force_login(user)
+            response = self.client.get(
+                reverse('admin:get_default_templates', args=[org1.pk])
+            )
+            self.assertEqual(response.status_code, 403)
+
+        with self.subTest('User requests data of other organization'):
+            org_owner = self._create_org_owner()
+            self.client.force_login(org_owner.organization_user.user)
+            response = self.client.get(
+                reverse('admin:get_default_templates', args=[org1.pk])
+            )
+            self.assertEqual(response.status_code, 403)
```

Comment:
this test is wrong, it passes also without your changes. Please write a failing test before you fix it and ensure your changes fix it.

Reply:
I added this test on current master and it failed. Can you please double check? 

I made the test more explicit. 

![Screenshot from 2021-04-07 15-57-55](https://user-images.githubusercontent.com/32094433/113852375-211a1200-97ba-11eb-863c-438b77228d39.png)